Add box2d 2.3.1 support

dmagunov opened the pull request:
Hello,

Pull request dedicated to this issue https://github.com/kripken/box2d.js/issues/64

kripken commented:
This looks good, thanks.

Not sure what you mean by "changes to catalog structure"?

dmagunov commented:
> Not sure what you mean by "changes to catalog structure"?

1. I've moved all js builds to the new folder "build".
2. Moved all demos "html5canvas, rube_testbed and webgl" to the new folder "demo"
3. Moved all helpers (DebugDraw) to the new folder "helpers". But maybe the better place for them is "buid/helpers". 

What do You think about that?

kripken commented:
Those all sound good. Thanks!
